frontend/packages/dev-console/src/components/monitoring/alerts/monitoring-silences.tsx

```diff
@@ -74,7 +74,7 @@ export const MonitoringSilencesPage: React.FC<{}> = () => (
   </ConnectedPage>
 );
 
-export const MonitoringSilenceDetailsPage: React.FC<{ match: any }> = ({ match }) => (
+export const MonitoringSilenceDetailsPage: React.FC<{ match: any }> = () => (
```

Comment:
Same here we can remove the Component type definition as we are not using the match anymore.